### ISSUE TITLE
[Usable – Make Code - Home page] Tooltip of ‘Help & More’ button is not visible via keyboard under ‘Home page’. 

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3545]
**User Experience**:
User rely on keyboard will find difficulty to interact with the tooltip information if  tooltip is not accessible through keyboard as user won't be able to perform functionality over such control.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify the tooltip of visual icon control ‘More’ by keyboard.

**Actual Result:**
Tooltip of ‘Help & More’ button is not visible via keyboard under ‘Home page’ and due to this, end user is unable to get the context of that visual icons and also won't be able to perform functionality over such controls.
Same issue exists at all controls present on 'Header' section.

**Expected Result:**
Tooltip of ‘Help & More’ button should be displayed via keyboard under ‘Home page’, so that User rely on keyboard will not find difficulty to interact with the tooltip information and perform functionality over such control.

**Note:**
This issue also exists with Firefox and Chrome Browser.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev4, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #Win10-Edge(Chromium)

[Usable_Tooltip is not visible of more icon.zip](https://github.com/microsoft/pxt-microbit/files/5473712/Usable_Tooltip.is.not.visible.of.more.icon.zip)
